src/main/webapp/index.html

```diff
@@ -48,10 +48,58 @@
           <button data-modal-close-button class="exit-button">&times;</button>
         </div>
         <div class="modal-body" id="results-body">
-          <div id="results-list-container"></div>
+          <div class="list" id="results-list-container"></div>
         </div>
         <div class="modal-body" id="reviews-body">
-          <div id="reviews-list-container"></div>
+          <div class="list" id="reviews-list-container"></div>
+        </div>
+        <div class="modal-body" id='rev-form-body'>
```

Comment:
Where did the rest of the modal body go? The elements that are populated for the search results and the review results.

Reply:
I'm not really sure what you mean by this? It populates well right now?

Reply:
Nevermind. Upon further inspection of the file I see my comment makes no sense. From the code snippet it showed me I thought you deleted code.